app/controllers/content_items_controller.rb

```diff
@@ -7,11 +7,18 @@ class ContentItemsController < ApplicationController
   def show
     load_content_item
     set_expiry
+    @ab_testing_group = ab_testing_group
     with_locale do
       render content_item_template
     end
   end
 
+  def should_present_b_view?
+    (ab_testing_group == "B") && new_navigation_enabled? && content_is_tagged_to_a_taxon?
+  end
+
+  helper_method :should_present_b_view?
```

Comment:
What do you think of changing this to `should_present_new_content_view`? It could be hard to understand what the B version is without more context.